rollup.config.js

```diff
@@ -24,6 +24,15 @@ const tsconfigOverride = {
   }
 };
 
+// when the build path is not a child of the root of the project path
+// TypeScript will output resources following the same path structure,
+// `BASEPATH` will be a relative path to the root of the source project which
+// we can use to determine what TypeScript would have output
+const basePathParts = process.env.BASEPATH.split(/[\\|\/]/);
```

Comment:
`process.env.PATH.split(path.delimiter);`

Reply:
I will try that, but the problem that I was chasing is that what gets passed from `BUILD.gn` in the argument doesn't respect the OS.  I _might_ actually be able to just get away with `"/"` because that is what `gn` seems to use on any platform.